opencanary/modules/__init__.py

```diff
@@ -85,8 +88,16 @@ def log(self, logdata, **kwargs):
             password = logdata.get("PASSWORD", None)
             if username or password:
                 data["honeycred"] = self.honeyCredHook(username, password)
-
-        self.logger.log(data)
+        
+        # Log only if not in ignorelist
+        notify = True
+        for ip in self.ignorelist:
+            if check_ip(peer.host,ip) == True:
```

Comment:
@cecio: where is peer defined?

Reply:
yeah, it's probably better to use `data['src_host']` since peer is defined at line `76` but it's under an if, so using the data array should be the right way here.
I'm going to modify this and make an additional commit.

Reply:
OK, just commited the change.
Let me know if you think additional modifications are needed. Thanks